static/src/stylesheets/module/identity/_wizard.scss

```diff
@@ -16,24 +16,45 @@
     right: 0;
     will-change: transform;
 }
+.js-off .manage-account-wizard__step {
+    border: 1px solid $neutral-8;
+    margin-bottom: $gs-gutter*4;
+    padding-bottom: $gs-gutter*4;
+}
 .manage-account-wizard__step--hidden {
     display: none;
 }
 .manage-account-wizard__step--out {
     display: block;
     position: absolute;
     animation: manage-account-wizard__fadeLeft .2s both;
+    @media (prefers-reduced-motion: reduce) {
```

Comment:
Good idea, but since it's only supported in Safari we have a global class, which get's added once a user disabled the a11y setting here: https://www.theguardian.com/help/accessibility-help

`.disable-flashing-elements`